codeserver/ubi9-python-3.9/Dockerfile

```diff
@@ -39,6 +39,24 @@ RUN yum install -y "https://github.com/coder/code-server/releases/download/${COD
 RUN chmod -R g+w /opt/app-root/lib/python3.9/site-packages && \
     fix-permissions /opt/app-root -P
 
+# Create a temporary directory for extensions
+WORKDIR /opt/app-root/extensions-temp
+
+COPY --chown=1001:0 utils/*.vsix /opt/vsix/
+
+# Install VS Code extensions
+RUN code-server --install-extension /opt/vsix/ms-python.python-2024.2.1.vsix --extensions-dir /opt/app-root/extensions-temp && \
+    code-server --install-extension /opt/vsix/ms-toolsai.jupyter-2023.9.100.vsix --extensions-dir /opt/app-root/extensions-temp
+
+# Ensure the temporary directory and target directory have the correct permissions
+RUN mkdir -p /opt/app-root/src/.local/share/code-server/extensions && \
+    mkdir -p /opt/app-root/src/.local/share/code-server/coder-logs && \
+    chown -R 1001:0 /opt/app-root/src/.local/share/code-server && \
+    chown -R 1001:0 /opt/app-root/extensions-temp && \
+    chown -R 1001:0 /opt/app-root/src/.config/code-server
```

Comment:
can you share why this is needed separately ?

maybe we should move line 39-40, to below, 
at line 121

Reply:
Moved